demo/src/main.ts

```diff
@@ -16,7 +15,7 @@ const mostReadableEl = document.querySelector<HTMLElement>('#mostReadable');
 const colorBoxEl = document.querySelector<HTMLElement>('#colorBox');
 
 function colorChange(color) {
-  const tiny = new TinyColor(color);
+  const tiny = new tinycolor.TinyColor(color);
```

Comment:
Please see the other comment first, but do we need to prefix with `tinycolor.` since TinyColor is exposed on the window in line 5?

Reply:
technically it might work, but typescript won't like it

Reply:
Alright, I was wondering if that was the case. No problem.